daprdocs/content/en/reference/api/workflow_api.md

```diff
@@ -90,6 +90,114 @@ Traceparent: 00-e3dedffedbeb9efbde9fbed3f8e2d8-5f38960d43d24e98-01
 Connection: close 
 ```
 
+## Pause workflow request
+
+Pause a running workflow instance with the given name and instance ID.
+
+```bash
+POST http://localhost:3500/v1.0-alpha1/workflows/<workflowComponentName>/<instanceId>/pause
+```
+
+### URL parameters
+
+Parameter | Description
+--------- | -----------
+`workflowComponentName` | Current default is `dapr` for Dapr Workflows
+`workflowName` | Identify the workflow type
```

Comment:
As per the comment above, I don't believe workflowName is necessary here nor the url parameters tables below. This also applies to Purge